examples/gno.land/r/demo/profile/profile_test.gno

```diff
@@ -0,0 +1,148 @@
+package profile
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/p/demo/testutils"
+)
+
+func TestStringFields(t *testing.T) {
+	addr := testutils.TestAddress("alice")
+	std.TestSetRealm(std.NewUserRealm(addr))
+
+	// Get before setting
+	name := GetStringField(addr, "DisplayName", "anon")
```

Comment:
Use constants for they keys

Reply:
https://github.com/gnolang/gno/pull/1983/commits/f7ff18c523e210d7ebac292bf6b0e3a4ff1d89c4